apps/meteor/ee/app/livechat-enterprise/server/jobs/AbstractOmniSchedulerClass.ts

```diff
@@ -0,0 +1,44 @@
+import Agenda from 'agenda';
+import { MongoInternals } from 'meteor/mongo';
+import { Collection } from 'mongodb';
+
+import { schedulerLogger } from '../lib/logger';
```

Comment:
Can we convert the logger in part of the class? That way subclasses can just use "this.debug" and so on for logging